common/djangoapps/student/tests/test_email.py

```diff
@@ -392,6 +392,7 @@ def assertChangeEmailSent(self, email_user):
             url = safe_get_host(request)
 
         self.assertIn(url, body)
+        return body
```

Comment:
A minor point: I don't think this needs to be added: no new code is using this assert.